validator/sawtooth_validator/server/cli.py

```diff
@@ -101,13 +105,20 @@ def main(args=sys.argv[1:]):
                           data_dir,
                           key_dir)
 
+    # pylint: disable=broad-except
     try:
         validator.start()
     except KeyboardInterrupt:
         print("Interrupted!", file=sys.stderr)
+        sys.exit(1)
     except LocalConfigurationError as local_config_err:
-        print(local_config_err, file=sys.stderr)
+        LOGGER.error(str(local_config_err))
+        sys.exit(1)
     except GenesisError as genesis_err:
-        print(genesis_err, file=sys.stderr)
-    finally:
-        validator.stop()
+        LOGGER.error(str(genesis_err))
+        sys.exit(1)
+    except SystemExit as e:
```

Comment:
This is thrown by `sys.exit()` and is not Caught by the next except block, since it doesn't inherit off of Exception.